### PR TITLE
Update copy on /support

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Updates based on [copydoc](https://docs.google.com/document/d/1AU2dwTXpQk2UuEvSuAqRKrbPWYb-LR0ZGvWu6xw95I8/edit#heading=h.pkvhhqabnqfc) changes and [design](https://www.figma.com/file/YdmmtZxcCdEVJE4JmSqfeD/Ubuntu.com%2Fsupport?type=design&node-id=233-3065&mode=design&t=teXUetDya3yo2y4d-0)

## QA

- Go to /support
- Check that the copy changes match
- Check the design matches

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10400

## Screenshots

image

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

**Comment by petesfrench** [comment]
@lyubomir-popov can you take a look please

File: templates/support/index.html [hunk 1]
```diff
@@ -640,12 +750,47 @@ <h3 class="p-heading--5">Modular MLOps platforms</h3>
...
+            <div class="col-3">
+              <p>Rely on <a href="/openstack">Canonical OpenStack</a> to build a cost-effective hybrid multi-cloud infrastructure with the peace of mind of experts on your side.</p>
+            </div>
+          </div>
+          <hr class="p-rule">
+          <div class="row">
+            <div class="col-3">
+              <div class="p-image-wrapper">
+                {{ image (
+                  url="https://assets.ubuntu.com/v1/eab36658-MicroCloud%20logo@2x.png",
```
Comment (carkod) [inline templates/support/index.html:776]: No "Canonical" word on top of the Microcloud logo? But up to design/UX

File: templates/support/index.html [hunk 2]
```diff
@@ -640,12 +750,47 @@ <h3 class="p-heading--5">Modular MLOps platforms</h3>
...
+                  alt="",
```
Comment (carkod) [inline templates/support/index.html:777]: I believe @juanruitina will ask us to fill this up.

File: templates/support/index.html [hunk 3]
```diff
@@ -640,12 +750,47 @@ <h3 class="p-heading--5">Modular MLOps platforms</h3>
...
         </div>
         <div class="col-6 col-medium-3">
-          <p>Rely on Canonical's <a href="/openstack">Charmed OpenStack</a> to build a cost-effective hybrid multi-cloud infrastructure with the peace of mind of experts on your side.</p>
-          <p>Want to deploy your clouds in minutes, with just a simple command? Check out Canonical's <a href="https://canonical.com/microcloud">Microcloud</a>, the low-touch cloud solution, optimized for easy and repeatable deployments.</p>
+          <div class="row">
+            <div class="col-3">
+              <div class="p-image-wrapper">
+                {{ image (
+                  url="https://assets.ubuntu.com/v1/6ac07c33-canonical-openstack.png",
+                  alt="",
```
Comment (carkod) [inline templates/support/index.html:758]: Probably need to fill this with the logo name.

**Review by carkod — APPROVED** [review]
LGTM, some minor comments.

File: templates/support/index.html (hunk 2 above)
Comment (juanruitina) [reply templates/support/index.html:777]: Yes! Can you please link both logos and add the alt text? People looove to click on images.

**Comment by petesfrench** [comment]
@lyubomir-popov Can you take a look at this please?

**Pull request merged**